docs/guides/using-org-slugs.mdx

```diff
@@ -0,0 +1,397 @@
+[active organization](/docs/organizations/overview#active-organization) at any given time. Clerk automatically sets organizations as
+active when a user picks an organization with a pre-built component (e.g. the
+[Organization Switcher](/docs/components/organization/organization-switcher)), and Clerk provides
+APIs (like [<code>auth()</code>](/docs/references/nextjs/auth) and [<code>useAuth()</code>](/docs/references/react/use-auth)), that let you determine the active organization and render the appropriate
+content.
+
+A relatively common practice is to represent organization-scoped spaces within an app by prefixing
+URLs with an organization slug. For example, imagine a B2B application named "Petstore" that has
+two customers, "Acmecorp" and "Widgetco". The application may choose to have urls like:
+
+`https://petstore.example.com/orgs/`<b>`acmecorp`</b>`/dashboard` ⬅ indicates **Acmecorp**'s dashboard
+`https://petstore.example.com/orgs/`<b>`widgetco`</b>`/dashboard` ⬅ indicates **Widgetco**'s dashboard
+
+It is also possible to use an [organization ID](/docs/references/javascript/organization/organization#properties) to
+indicate the active organization in URLs:
+
+`https://petstore.example.com/orgs/`<b>`org_1a2b3c4d5e6f7g8e`</b>`/dashboard` ⬅ **Acmecorp's ID** indicates Acmecorp's dashboard
+
+This guide will focus on using a human-comprehensible slug, but IDs are just as well-supported in Clerk's component
+configuration.
+
+> [!CAUTION]
+> Think carefully about whether you need to include organization slugs in URLs! Consider:
+>
+> - Will organization-specific links be more useful to your users than
+>   organization-agnostic links? If most users will only ever be in one organization (that represents
+>   their company), they won't need the slug to disambiguate the organization when sharing
+>   links with their coworkers. Public documentation, marketing, and third-party blogs are also
+>   made easier to write if links aren't tied to any specific organization.
+> - Adding an additional piece of state (i.e. the URL) that indicates the organization will require additional
+>   complexity in your application, as opposed to managing it soley and transparently in the Clerk
+>   session.
+>
+> We at Clerk generally recommend against including organization slugs in URLs. If your application
+> does have this requirement though, this guide will cover the best practices for doing so with a NextJS application.
+
+<Steps>
+  ### Defining URL patterns
+
+  To begin, you can define which sections of your application are scoped to organizations and which belong to
+  the [personal account](https://clerk.com/glossary#personal-account). For this example, we'll assume that the prefix
+  `/orgs/` indicates that an organization is active, followed by the organization slug, and the prefix `/me/` indicates that the
+  personal account is active.
+
+  | URL | What should be active? | What should be displayed? |
```

Comment:
Can you elaborate a bit here? What are examples of what would be returned by the `auth` helper?

With this section, i'm attempting to define the application's requirements in a fairly clerk-agnostic way, so that I can then show how it can be modeled using clerk.

Reply:
I meant what's returned as the `orgSlug` from `auth`. Not necessary this would be used within a server component to be displayed, it could also be used within a server action. 

But I understand your goal here and it also looks good to me since this guide is mostly components-based, feel free to go according to your preference.